OSDOCS-16948#CQA work Stor4 - vSphere CSI driver (#114546)
openshift/openshift-docs: modules/persistent-storage-csi-vsphere-adding-bm-nodes.adoc

= Adding bare-metal nodes

[role="_abstract"]

> {product-title} supports adding bare-metal nodes to a cluster on vSphere as a Technology Preview feature. 

Reviewer: ```suggestion
{product-title} has the ability to add bare-metal nodes to a cluster on vSphere as a Technology Preview feature. 
```
To be safe.
https://redhat-documentation.github.io/supplementary-style-guide/#technology-preview-guidance
